core/src/edu/asu/c3simulator/widgets/SellCompany.java

```diff
@@ -0,0 +1,87 @@
+package edu.asu.c3simulator.widgets;
```

Comment:
I think I reviewed this above. Get rid of the duplicates, and make sure they are identical.